package.json

```diff
@@ -24,10 +24,19 @@
   },
   "homepage": "https://github.com/Gamerman2001/discord-audio-bot#readme",
   "dependencies": {
+    "@discordjs/opus": "^0.5.0",
     "discord.js": "^12.5.3",
-    "dotenv": "^8.2.0"
+    "dotenv": "^8.2.0",
```

Comment:
is this one a dep or a dev dependency

Reply:
Honestly I'm not sure. Dotenv is needed so I can grab variables located in my env file..... I googled it and some say leave it as a regular dependency. The Docs install it as a regular dependency as well. I'm going to lean more on regular dependency on this one because my app won't function without calling this library.

Reply:
> my app won't function without calling this library

Yep that's the definition of a dependency! 

A dev dependency is something like ESLint or typescript types files - things that your app doesn't need after it compiles.